lib/services/consumptionManagement/lib/consumptionManagementClient.js

```diff
@@ -1,8 +1,4 @@
 /*
- * Copyright (c) Microsoft Corporation. All rights reserved.
- * Licensed under the MIT License. See License.txt in the project root for
- * license information.
- *
```

Comment:
if you are using autorest to generate the client manually then please use
```
--license-header=MICROSOFT_MIT_NO_VERSION
```

also make sure to use `--azure-arm`

Reply:
just noticed your message. I'll make the change now.